ocrd_models/README.md

```diff
@@ -2,48 +2,50 @@
 
 > OCR-D framework - file format APIs and schemas
 
-See https://github.com/OCR-D/core
+## Adding user methods to the generated OcrdPage API
 
-## Adding user methods to the generated PAGE API
-
-Let's say you want to add a method `get_FirstTextRegion` on the pc:Page element:
+Let's say you want to add a method `get_FirstTextRegion` on the `pc:Page` element:
 
 1. Create a file `ocrd_models/ocrd_page_user_methods/get_FirstTextRegion.py`
 
-```python
-def get_FirstTextRegion(self):
-  return self.get_TextRegion[0]
-```
+    ```python
+    def get_FirstTextRegion(self):
+        return self.get_TextRegion[0]
+    ```
 
 2. Edit `ocrd_models/ocrd_page_user_methods.py` and append to the `METHOD_SPECS` list:
 
-```python
-METHOD_SPECS = (
-  # ...
-  _add_method(r'^PageType$', 'get_FirstTextRegion')
-  # ...
-)
-```
+    ```python
```

Comment:
Why the additional indentation?

Reply:
Ah, of course, I was only watching the changed README instead of the original state :)